src/js/modules/infragistics.ui.videoplayer.js

```diff
@@ -49,223 +49,1826 @@
 		},
 
 		options: {
-			/* type="array" Get or set a list of video sources to choose from. Best coded/format is automatically detected by the control. Supported types are depending on the browser and could be one of the following mov, mp4, webm, ogg. */
+			/* type="array" Get or set a list of video sources to choose from. Best coded/format is automatically detected by the control. Supported types are depending on the browser and could be one of the following mov, mp4, webm, ogg.
+			```
+				//Initialize
+				$(".selector").igVideoPlayer({
+					sources: [
+						"videos/QuinceIntro_1.h264.mp4",
```

Comment:
The code snippet should be runnable. Don't use local sources. 

'''
sources: [
	"http://dl.infragistics.com/pg/2011-1/web/shared/videoplayer/videos/QuinceIntro_Part3_1.h264.mp4",
	"http://dl.infragistics.com/pg/2011-1/web/shared/videoplayer/videos/QuinceIntro_Part3_1.webmvp8.webm",
	"http://dl.infragistics.com/pg/2011-1/web/shared/videoplayer/videos/QuinceIntro_Part3_1.theora.ogv"
]
'''